networks/suzuka/suzuka-full-node/src/manager.rs

```diff
@@ -27,6 +29,27 @@ impl Manager<SuzukaPartialNode<Executor>> {
     }
 
     pub async fn try_run(&self) -> Result<(), anyhow::Error> {
+
+        let (stop_tx, mut stop_rx) = tokio::sync::watch::channel(());
+        tokio::spawn({
+            let mut sigterm = signal(SignalKind::terminate()).context("Can't register to SIGTERM.")?;
+            let mut sigint = signal(SignalKind::interrupt()).context("Can't register to SIGKILL.")?;
+            let mut sigquit = signal(SignalKind::quit()).context("Can't register to SIGKILL.")?;
```

Comment:
I could offer the same comments here as with #191, but let's address this in one place.

Reply:
Will do.